geography-author/README.md

```diff
@@ -0,0 +1,196 @@
+# Mobility Data Specification: Geography Author
+
+This specification contains a collection of RESTful APIs used to define how to create, update, delete, publish, and read Geographies and GeographyMetadata.
+
+- Authors: LADOT
+- Date: 21 February 2020
+- Version: beta
+
+## Table of Contents
+
+- [Audience](#audience)
+- [Background](#background)
+- [Distribution](#distribution)
+- [Schema](#schema)
+- [File Format](#file-format)
+- [Endpoints](#endpoints)
+
+
+<a name="background"></a>
+
+## Background
+
+The main intended clients of this API are the Policy, Policy Author, and Jurisdiction services. A Policy object may require
```

Comment:
I wouldn't say these are the clients of the API, they aren't actually using the API at all.